webapp/platform/client/src/websocket.ts

```diff
@@ -71,7 +82,9 @@ export default class WebSocketClient {
     private serverHostname: string | null;
     private postedAck: boolean;
 
-    constructor() {
+    private reconnectTimeout: ReturnType<typeof setTimeout> | null;
```

Comment:
This is a pet peeve of TS, but if you change the `setTimeout` call below to `window.setTimeout`, you can type this as `number | null`. For some reason, the types we have confuse Node's `setTimeout` (which we don't use as a web app except maybe in unit tests) with the browser's `setTimeout` (which we do use)

Reply:
I was going to suggest that you could change line 193 to `this.reconnectTimeout = window.setTimeout(`, but I just realized that, while that would probably work in the browser, I'm betting it would cause problems when this code is used from a Node environment since there's a small chance it'd behave differently.

Given this is just a quirk with some minor typing differences between the web and Node, let's just leave it as-is